spider_cms.py

```diff
@@ -627,10 +626,45 @@ def main(args):
     fd.close()
     os.rename(tmpname, "checkpoint.json")
 
-    logging.warning("Total processing time: %.2f mins" % (
+    logging.warning("Total processing time for history: %.2f mins" % (
                     (time.time()-starttime)/60.))
 
 
+    # Now that we have the fresh history, process the queues themselves.
+    if not args.process_queue: return 0
+
+    pool = multiprocessing.Pool(processes=10)
```

Comment:
Reuse the same pool instead of creating a new one.

Reply:
Done. I also had to remove the `pool.close()` and `pool.join()` after the history phase, otherwise I can't add any new processed to the pool. I figure it's ok to just remove those calls (as opposed to calling them `if args.process_queue`) since we explicitly check for timed out processed and call `pool.terminate()` in case. If the queues don't get processed, the pool is automatically terminated when exiting main. Let me know if I see that wrongly.